src/core/EditableTable.tsx

```diff
@@ -120,6 +120,39 @@ const EditableTable = <T extends object>(props: IProps<T>) => {
     const previous = usePrevious(props.data)
     const addButtonColor = (props.color !== 'disabled' && props.color) || 'primary'
 
+    const localization = {
+        body: {
+            dateTimePickerLocalization: { locale: ptBRLocale },
+            emptyDataSourceMessage:
+                'Não há dados para serem exibidos no momento',
+            editRow: {
+                saveTooltip: 'Salvar',
+                cancelTooltip: 'Cancelar',
+                deleteText:
+                    'Você tem certeza que deseja excluir esse '
+                    + props.title + '?'
+            },
+            addTooltip: 'Adicionar ' + props.title,
+            deleteTooltip: 'Remover ' + props.title,
+            editTooltip: 'Editar ' + props.title
```

Comment:
You could use string interpolation with backticks (`) here 👀

Reply:
agreed

Reply:
Thanks for your suggestion, Mr. @haskellcamargo. We will fix it on next PR.